contracts/eden/src/inductions.cpp

```diff
@@ -435,6 +435,35 @@ namespace eden
       return pos != invitee_idx.end() && pos->invitee() == invitee;
    }
 
+   void inductions::on_rename(eosio::name old_account, eosio::name new_account)
```

Comment:
Doesn't update `endorsement::inviter`. It's probably ok to assert if `induction::inviter` exists, since it indicates recent activity.

Reply:
I missed this one because `endorsement::inviter` not even used anywhere.